Cargo.toml

```diff
@@ -10,32 +10,33 @@ default-members = ["cmd/soroban-cli"]
 [workspace.dependencies.soroban-env-host]
 version = "0.0.12"
 git = "https://github.com/stellar/rs-soroban-env"
-rev = "993c527abce72748405d71467498bffd63e061c1"
+rev = "2431bd416010215cc9759b435662a7274206a4c0"
 
 [workspace.dependencies.soroban-spec]
 version = "0.4.1"
 git = "https://github.com/stellar/rs-soroban-sdk"
-rev = "e1c3de33942f0e997680645941787102ebf61e85"
+rev = "f2f4bf0e8197ee43fd6242c4a31943f466815bed"
 
 [workspace.dependencies.soroban-token-spec]
 version = "0.4.1"
 git = "https://github.com/stellar/rs-soroban-sdk"
-rev = "e1c3de33942f0e997680645941787102ebf61e85"
+rev = "f2f4bf0e8197ee43fd6242c4a31943f466815bed"
 
 [workspace.dependencies.soroban-sdk]
 version = "0.4.1"
 git = "https://github.com/stellar/rs-soroban-sdk"
-rev = "e1c3de33942f0e997680645941787102ebf61e85"
+rev = "f2f4bf0e8197ee43fd6242c4a31943f466815bed"
```

Comment:
@tamirms , these sdk refs will bump to v0.5.0 tag correct?
https://github.com/stellar/rs-soroban-sdk/releases/tag/v0.5.0